zetaclient/context/app.go

```diff
@@ -127,6 +140,22 @@ func (a *AppContext) GetCrossChainFlags() observertypes.CrosschainFlags {
 	return a.crosschainFlags
 }
 
+// SetRelayerKeyPasswords sets the relayer key passwords for given networks
+func (a *AppContext) SetRelayerKeyPasswords(relayerKeyPasswords map[chains.Network]string) {
+	a.mu.Lock()
+	defer a.mu.Unlock()
+
+	a.relayerKeyPasswords = relayerKeyPasswords
+}
+
+// GetRelayerKeyPassword returns the relayer key password for the given network
+func (a *AppContext) GetRelayerKeyPassword(network chains.Network) string {
```

Comment:
This should be a part of AppContext's `Chain{}`. 

Example: `chain.RelayedKeyPassword() (string, bool) // bool == exists`

Reply:
Passwords are static values passed by user (per network, not per chain_id) while chains are dynamically added/removed.

Reply:
Can you elaborate why we do it per network? Do we have this feature for btc or evm chains?

Reply:
I propose the following:

```go
func (c Chain) RelayerKeyPassword() (string, bool) {
  networkID := chain.RawChain().Network
  v, found := c.registry.relayerKeyPasswords[networkID]

  return v, found
}
```

Reply:
Why, per network?
Answer: operator could use one `solana.json` key as the relayer key for any Solana network (testnet, devnet, mainnet), the key format is same.

We don't have this for EVM/btc chain. It's an optional key for non-ECDSA signature chains (Solana, TON, etc.), operator could skip by providing empty passwords if they're not a relayer.

Reply:
I can move the password map to registry and expose the getter from `Chain` object. Not a problem.

Reply:
password map is moved to chain registry: https://github.com/zeta-chain/node/pull/2673/commits/6a4399ce1b26b4c772398a5c69895a63f2c47632